draft_editor/draft_editor.py

```diff
@@ -0,0 +1,67 @@
+from talon import Module, actions, ui
+import time
+
+mod = Module()
+mod.mode("draft_editor", "Indicates whether the draft editor has been activated")
+
+active_window = None
+
+@mod.action_class
+class Actions:
+    def draft_editor_open():
+        """Open draft editor"""
+        global active_window
+        active_window = ui.active_window()
+        editor = get_editor()
+        has_selected_text = actions.edit.selected_text() != ""
+        if has_selected_text:
+            actions.edit.copy()
```

Comment:
I don't know if this is "better" per se, but maybe use `clip.set_text` since we've already got the text.

```
selected_text = actions.edit.selected_text()

if selected_text != ""
   actions.clip.set_text(selected_text)
```

Reply:
That would definitely work but what would be the upside? Is using clip.set_text more efficient than doing just a copy?

The third option would be just to do.
```
if selected_text != "":
    actions.user.paste(selected_text)
```

Reply:
I like your third option, good call.

Reply:
Pushed this change.